indexer-api/graphql/schema-v3.graphql

```diff
@@ -34,6 +34,10 @@ type Block {
 	The hex-encoded ledger parameters for this block.
 	"""
 	ledgerParameters: HexEncoded!
+	"""
+	The system parameters (governance) at this block height.
```

Comment:
At which block height? All other comments refer to "block", so better replace with "for this block".
(actually this needs to be changed below)